docs/user-guide/authenticating-with-client-certificates.md

```diff
@@ -0,0 +1,75 @@
+# Authenticating with client certificates
+
+You can perform authentication with client certificates.
+
+If the keyring or a truststore contains at least one valid certificate authority (CA) other than the CA of API ML, it is possible to use the client certificates issued by this CA to authenticate to API ML. This feature is not enabled by default and must be configured.
+
+:::note
+There is a limitation with respect to ACF2 systems. If you would like to offer feedback using client
+certificate authentication, please create an issue against the api-layer repository.
+:::
+
+When providing credentials with a client certificate on the same login request, the credentials take precedence and the client certificate is ignored.
```

Comment:
This is true only for login request, otherwise the basic authentication is ignored by API ML.